app-service-environment-serviceplan-logicapp/azuredeploy.json

```diff
@@ -8,6 +8,12 @@
         "description": "The name to use for the new AppService Environment"
       }
     },
+    "location": {
```

Comment:
Yeah I tried a vnet in the RG and outside the RG (in the same region) but never got a successful deployment of this template.